### PR TITLE
Introduction of dubbo2.7 new features 

### PR DESCRIPTION
**Pull request by lexburner**
1. new a blog to introduce dubbo2.7 new features 
2. format blog.js 
3. change my Chinese name to English nickname"

### CODE REVIEW

File: blog/zh-cn/dubbo-27-features.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    AsyncService.sayHello("Han Meimei");
+    Future<String> fooFuture = RpcContext.getContext().getFuture();
+    fooFuture.get();
+    ```
+
+可以看出，这样的使用方式，不太符合异步编程的习惯，竟然需要从一个上下文类中获取到 Future。如果同时进行多个异步调用，使用不当很容易造成上下文污染。而且，Future 并不支持 callback 的调用方式。这些弊端在 Dubbo 2.7 中得到了改进。
+
+### 4.3 Dubbo 2.7 异步化
+
+1. 无需配置中特殊声明，显示声明异步接口即可
```
Comment (kezhenxu94) [inline blog/zh-cn/dubbo-27-features.md]: 显示 => 显式?

File: blog/zh-cn/dubbo-27-features.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+Q：如果 RPC 接口只定义了同步接口，有办法使用异步调用吗？
+
+A：2.6 中的异步调用唯一的优势在于，不需要在接口层面做改造，又可以进行异步调用，这种方式仍然在 2.7 中保留；使用 Dubbo 官方提供的 compiler hacker，编译期自动重写同步方法，请[在此](https://github.com/dubbo/dubbo-async-processor#compiler-hacker-processer)讨论和跟进具体进展。
+
+---
+
+Q：关于异步接口的设计问题，为何不提供编译插件，根据原接口，自动编译出一个 XxxAsync 接口？
+
+A：Dubbo 2.7 采用采用过这种设计，但接口的膨胀会导致服务类的增量发布，而且接口名的变化会影响服务治理的一些相关逻辑，改为方法添加 Async 后缀相对影响范围较小。
```
Comment (kezhenxu94) [inline blog/zh-cn/dubbo-27-features.md]: 采用采用 => 采用?

**Comment by lexburner** [comment]
@kezhenxu94 fixed

File: blog/zh-cn/dubbo-27-features.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+- 异步化改造
+- 三大中心改造
+- 服务治理增强
+
+## 4 异步化改造
+
+### 4.1 几种调用方式
+
+![调用方式](http://kirito.iocoder.cn/image-20190321160844133.png)
```
Comment (lixiaojiee) [inline blog/zh-cn/dubbo-27-features.md]: I suggest putting the pictures in the project directory

Comment (lexburner) [reply blog/zh-cn/dubbo-27-features.md]: My other article also relies on my own domain.

Comment (lixiaojiee) [reply blog/zh-cn/dubbo-27-features.md]: ok

Comment (ralf0131) [reply blog/zh-cn/dubbo-27-features.md]: I am +1 on putting the pictures in the project directory. Actually shall we really need this picture?

**Pull request merged**